pkg/operator/starter.go

```diff
@@ -139,12 +140,31 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		ctx.EventRecorder,
 	)
 
+	certRotationBase := time.Duration(0)
+	err = wait.PollImmediate(time.Second, 1*time.Minute, func() (bool, error) {
+		certRotationConfig, err := kubeClient.CoreV1().ConfigMaps(operatorclient.GlobalUserSpecifiedConfigNamespace).Get("unsupported-cert-rotation-config", metav1.GetOptions{})
+		if err != nil {
+			if errors.IsNotFound(err) {
+				return true, nil
```

Comment:
is this just retrying for connection errors?

Reply:
That is correct.